js/ui/editors/ConstructorControlPanel.js

```diff
@@ -319,20 +324,71 @@ export class ConstructorControlPanel extends ConstructorEditorView {
             projectProperties.config_name_set
         );
 
+        UIPreferences.addDropdown(
+            this.projectSettings,
+            'Runner Type',
+            projectProperties.runner_project_get() ?? 'Use Default',
```

Comment:
Not a fan of how much of a sentinel value `'Use Default'` is getting, I'd prefer to store it somewhere I think just so there's no possible typing error :P